docs/pages/docs/features/caching.mdx

```diff
@@ -75,6 +75,48 @@ You can also always disable caching on a specific task by setting `cache` to `fa
 }
 ```
 
+## Alter Caching Based on Environment Variables
+
+You can control caching based on the value of environment variables by setting `env` key where applicable.
+
+**At the root** will tell `turbo` to include the hashed values of each environment variable it all other hashes.
```

Comment:
As non native speaker I find this confusing wording, I find the bit 'the hashed values of each environment variable it all other hashes' confusing.

Reply:
I will here repeat the example that's given on a different page, this one:

>  - The value of the GITHUB_TOKEN environment variable will alter the hash of all tasks.
>  - The values of GITHUB_TOKEN and SOMETHING_ELSE environment variables will alter the hash of all build tasks
>  - The values of GITHUB_TOKEN, STRIPE_SECRET_KEY, and NEXT_PUBLIC_STRIPE_PUBLIC_KEY environment variables will alter the hash of only the build task for a package/app named web
>